cmd/cmd.go

```diff
@@ -85,9 +85,8 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 			path := modelfile.Commands[i].Args
 			if path == "~" {
 				path = home
-			} else if strings.HasPrefix(path, "~/") {
-				path = filepath.Join(home, path[2:])
 			}
+			path = strings.TrimPrefix(path, "~/")
```

Comment:
TrimPrefix returns unchanged string if not applicable.
Ref: https://pkg.go.dev/strings#TrimPrefix